templates/login.html

```diff
@@ -1,14 +1,10 @@
 {% extends "base.html" %}
 {% load bootstrap4 %}
-{% block title %}
-Login Here
-{% endblock %}
-
 {% block content %}
 <h2>Log In</h2>
 <form method="post">
-  {% csrf_token %}
-  {{ form.as_p }}
-  <button type="submit">Log In</button>
+ {% csrf_token %}
+ {{ form.as_p }}
```

Comment:
Here's the fix for the forms so they use bootstrap (double check this and then commit the result from your computer if this looks right)
```suggestion
 {% bootstrap_form form %}
```